homeassistant/components/simplisafe/.translations/en.json

```diff
@@ -10,13 +10,23 @@
         "step": {
             "user": {
                 "data": {
-                    "code": "Code (for Home Assistant)",
                     "password": "Password",
                     "username": "Email Address"
                 },
                 "title": "Fill in your information"
             }
         },
         "title": "SimpliSafe"
+    },
+    "options": {
+        "step": {
+            "init": {
+                "data": {
+                    "code": "Code (used in Home Assistant UI)"
+                },
+                "description": "Set various options for the AirVisual integration.",
```

Comment:
?

Step descriptions are optional for options. If you don't have any, just omit it.